src/Compilers/CSharp/Portable/Lowering/ClosureConversion/ClosureConversion.cs

```diff
@@ -1129,7 +1129,7 @@ private BoundBlock RewriteBlock(BoundBlock node, ArrayBuilder<BoundExpression> p
 
             if (prologue.Count > 0)
             {
-                newStatements.Add(new BoundSequencePoint(null, null) { WasCompilerGenerated = true });
+                newStatements.Add(new BoundSequencePoint(null!, null) { WasCompilerGenerated = true });
```

Comment:
>new BoundSequencePoint(null! [](start = 34, length = 28)

same question (why not make the parameter nullable?)

Reply:
A bound sequence point is a special case as the only node that permits a null syntax.  There is no easy way to annotate the API to permit that.

---
In reply to: [386591092](https://github.com/dotnet/roslyn/pull/42068#discussion_r386591092) [](ancestors = 386591092)